src/jobs/deploy.yml

```diff
@@ -0,0 +1,22 @@
+description: >
+  Deploy to Heroku
+
+  This job is intended to be used by projects that deploy to a single
+  application.
+
+parameters:
+  app:
+    description: Heroku's application name
+    type: string
+
+docker:
+  - image: carwow/ruby-ci:2.7-node-12
```

Comment:
Should we make this tag configurable as well?
Help us to gradually update different projects in the future?

Reply:
I think for that we can just bump a new orb version, but a parameter with a default value can also be added later if needed.

Reply:
Added an executor which is the new way of doing this. ;)